cristin-project/src/test/java/no/unit/nva/cristin/projects/query/organization/QueryCristinOrganizationProjectHandlerTest.java

```diff
@@ -131,6 +143,42 @@ void shouldReturnSearchResponseWithEmptyHitsWhenBackendFetchIsEmpty() throws Api
         assertThat(0, equalTo(searchResponse.getHits().size()));
     }
 
+    @Test
+    void shouldAddParamsToCristinQueryForFilteringAndReturnOk() throws IOException, ApiGatewayException {
+        cristinApiClient = spy(cristinApiClient);
+        doReturn(new HttpResponseFaker(EMPTY_LIST_STRING,
+                HttpURLConnection.HTTP_OK, generateHeaders(ZERO_VALUE, LINK_EXAMPLE_VALUE)))
+                .when(cristinApiClient).listProjects(any());
+        handler = new QueryCristinOrganizationProjectHandler(cristinApiClient, new Environment());
+        var queryParams = Map.of("funding", FUNDING_SAMPLE,
+                "biobank", BIOBANK_SAMPLE,
+                "keyword", KEYWORD_SAMPLE,
```

Comment:
For later, consider also using constants for the keys, i think you already have defined them here: https://github.com/BIBSYSDEV/nva-cristin-service/blob/main/cristin-commons/src/main/java/no/unit/nva/cristin/model/JsonPropertyNames.java